how-pyth-works/cross-chain.md

```diff
@@ -0,0 +1,16 @@
+# Cross-chain
+
+Pyth needs a cross-chain component to ferry prices on Pythnet to target chains. The diagram below shows how prices are delivered from Pythnet to target chains.
+
+![](<../.gitbook/assets/pythnet_schema.png>)
```

Comment:
Nitpick : some parts of the schema are not visible because of the dark theme in the pyth docs.
https://docs.pyth.network/~/revisions/NdVGaH8lDYfRE7tRQTAE/how-pyth-works/cross-chain

Reply:
i'll ask the design team